pydatalab/src/pydatalab/routes/v0_1/collections.py

```diff
@@ -419,3 +419,36 @@ def add_items_to_collection(collection_id):
         )
 
     return (jsonify({"status": "success"}), 200)
+
+
+@COLLECTIONS.route("/collections/<collection_id>/items", methods=["DELETE"])
+def remove_items_from_collection(collection_id):
+    data = request.get_json()
+    item_ids = data.get("item_ids", [])
+
+    collection = flask_mongo.db.collections.find_one(
+        {"collection_id": collection_id, **get_default_permissions()}
```

Comment:
```suggestion
        {"collection_id": collection_id, **get_default_permissions()}, project={"_id": 1}
```

Can just project the underlying ID to avoid returning a load of extra info from the db